### PR TITLE
2170 project description

### PR DESCRIPTION
**Pull request by marcellmueller**
Implements #2170

### CODE REVIEW

**Comment by marcellmueller** [comment]
@AntBush thanks again for the heads about about the internal project description. This is updated whenever you have time!

File: app/schema/mutations/application/createNewFormData.ts [hunk 1]
```diff
@@ -7,7 +7,14 @@ const mutation = graphql`
   mutation createNewFormDataMutation($input: CreateNewFormDataInput!) {
     createNewFormData(input: $input) {
       formData {
+        id
```
Comment (AntBush) [inline app/schema/mutations/application/createNewFormData.ts:10]: Do we still need this change now that we're not creating a new form data?

**Review by AntBush — APPROVED** [review]
Works well, just a small but inconsequential nit

File: app/schema/mutations/application/createNewFormData.ts (hunk 1 above)
Comment (marcellmueller) [reply app/schema/mutations/application/createNewFormData.ts:10]: No I'll remove it thanks!

**Pull request merged**